src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -469,6 +469,12 @@ protected override Encoding StandardOutputEncoding
             get { return (string?)_store[nameof(LangVersion)]; }
         }
 
+        public bool ReportIVTs
```

Comment:
Tagging @billwagner as FYI. This PR adds a new compiler option, which would need to be documented in [https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/) (and the VB counterpart)

Reply:
https://github.com/dotnet/docs/issues/35921 tracks documenting it.